myhpi/templates/base.html

```diff
@@ -76,13 +81,20 @@
                 <div id="nav-level-2" class="nav-level"></div>
 
                 <div id="nav-level-right"
-                    class="nav-level d-flex flex-column flex-grow-1 align-items-stretch align-items-xl-end">
+                    class="nav-level d-flex flex-column flex-grow-2 align-items-stretch align-items-xl-end">
                     <ul id="nav-item-container-user" class="nav-item-container collapse"
                         data-navbar-level="#nav-level-0">
+                        {% if user.groups.all.exists %}<span class="nav-bar-group-header"><i>Deine Gruppen:</i></span>
+                            {% for group in user.groups.all %}
+                                <li class="nav-bar-group-item">{{ group }} </li>
+                                {% endfor %}
+                            <li class="dropdown-divider"></li>
+                        {% endif %}
                         <li class="nav-item">
-                            <a class="nav-link" role="button">
-                                Abmelden
-                            </a>
+                            <form action="{% url 'oidc_logout' %}" method="post" id="logout-form">
+                                {% csrf_token %}
+                                <button type="submit" class="dropdown-item" href="/">Logout</button>
```

Comment:
Due to this being a button, the styling is completely off. Imo, the easiest solution is to use the link that was there before instead of the button tag and submit the form via JS like this: https://stackoverflow.com/questions/4286466/use-a-normal-link-to-submit-a-form

Reply:
would just using the bootstrap `<button type="button" class="btn btn-link">` work? I'd really prefer not doing this in JS

Reply:
Would you mind sharing in which browser the styling seems off? I've tried to replicate the behaviour, however in Chromium based browsers, Safari and Firefox the styling is fine on my end.

Reply:
@ksturtzkopf In basically every browser (I am using Firefox). Could you run `python manage.py collectstatic` and reload the page? And then send in a screenshot of how it looks like?

Reply:
On my end it looks like this (after running `python manage.py collecstatic`) 

<img width="1328" alt="Screenshot of myHPI landing page" src="https://user-images.githubusercontent.com/24505982/200576427-29185394-6678-4547-a56f-d4b73cd89ea8.png">

Reply:
Might be the same as for me. Press the Logout button and have a look, whether it lights up in orange
![grafik](https://user-images.githubusercontent.com/44174681/200578468-b8c09bdd-be92-4a60-b8ca-c493f9421ba0.png)

Compare this to the look (e.g. colors and animations) from the other menu items
![grafik](https://user-images.githubusercontent.com/44174681/200578614-de2af58f-ab6f-48ae-a110-0748e107b948.png)

Reply:
yes, the button does light up in orange when pressed. 

_However_, this appears to be the default behaviour for `dropdown-items`. On my end at least.

Reply:
May be. But our menu does not use `dropdown-items` from Bootstrap

Reply:
Compare with the other nav-items

Reply:
We *could* start using `dropdown-items` in the whole menu (and then restyle the Bootstrap `dropdown-item` classes). But on the other hand the menu is no dropdown in the bootstrap sense, so I am not sure whether we should really do this

Reply:
I reckon it would make more sense to implement the desired behaviour for these items explicitly.
I agree that using JS is the more favourable solution (it was also the original implementation ;) ), but I fully understand why @jeriox prefers using plain HTML.
So, where do we go from here? 
I'd be happy to revert back to using JS and implement the styling using the classes available.

Reply:
(and just as an fyi: Not using `dropdown-items` does result in the button being displayed as a button and not in the style we want to achieve ;) )

Reply:
>I reckon it would make more sense to implement the desired behaviour for these items explicitly.
Sorry, I what do you mean with "implementing explicitly"?

>(and just as an fyi: Not using `dropdown-items` does result in the button being displayed as a button and not in the style we want to achieve ;) )
As far as I can see neither adding nor omitting `dropdown-items` results in the style of the other menu items (but yes, adding `dropdown-items` makes it harder to spot)

Reply:
As I would like to wrap this PR up: @jeriox @SilvanVerhoeven, I am more than happy to implement the Logout Routing in JS or leave it as a form. 
To prevent having large discussions after committing to one solution, I would like to ask you to mediate between you two as it seems that you have the strongest opinions on the matter.

Reply:
> Sorry, I what do you mean with "implementing explicitly"?

Perhaps that wasn't the best word to describe what I meant. 
To ensure that the design is matched I would restyle the dropdown menu to match the desired style and indentation without using the bootstrap components. (i.e. in `nav-link`)

Reply:
@SilvanVerhoeven convinced me that doing it without JS is too ugly and cumbersome regarding the design, so feel free to go ahead and implement it in JS :)